src/OpenTelemetry.Api/Context/BaggageContext.cs

```diff
@@ -0,0 +1,241 @@
+﻿// <copyright file="BaggageContext.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace OpenTelemetry.Context
+{
+    /// <summary>
+    /// Baggage context.
```

Comment:
nit: Add a link to OTel spec here.

Reply:
Done. It's a dead link right now, but should work once the rename PR is merged.